website/docs/wallet/withdraw-validator.md

```diff
@@ -16,87 +18,68 @@ import TabItem from '@theme/TabItem';
 
 :::caution Public Preview
 **This feature is currently in public preview** and may change significantly as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) to share your feedback.
-Withdrawals are not yet available as a feature on Ethereum mainnet, and are currently only available on development test networks for trial at the time of writing (February 1st, 2023). The instructions in this document WILL NOT work on mainnet and are subject to change by the time withdrawals are available on mainnet.
+
+Note that withdrawals **aren't yet available on Ethereum mainnet**; this functionality is currently being validated on test networks. The instructions in this document won't work on Ethereum mainnet and may change significantly as the feature is stabilized on test networks.
 :::
 
-## Background
 <img  src={CapellaWithdrawalPng} /> 
-In the **Capella/Shanghai Ethereum** upgrade, stakers will be able to withdraw their validator(s)’ funds from staking on the network. When withdrawing, stakers can either choose to 
 
-(a) **partially withdraw** their **earnings**, that is, all value staked above 32 ETH, and continue validating
+The **Capella/Shanghai Ethereum** upgrade lets you withdraw your validator nodes' staked Ethereum in one of two ways: 
```

Comment:
This document is quite long, so I'm taking some liberties here in terms of removing unnecessary headers and tightening up the content as much as possible. This will be an ongoing iterative process, no need to get it perfect right now.